core/src/components/badge/test/hint/badge.e2e.ts

```diff
@@ -21,4 +21,22 @@ configs({ directions: ['ltr'], modes: ['md', 'ios', 'ionic-md'] }).forEach(({ co
       await expect(container).toHaveScreenshot(screenshot(`badge-hint-avatar`));
     });
   });
+
+  test.describe(title('badge: hint inside tab button'), () => {
+    test('tab button icon top, should not have visual regressions', async ({ page }) => {
+      await page.goto('/src/components/badge/test/hint', config);
+
+      const container = page.locator('#tab-button');
+
+      await expect(container).toHaveScreenshot(screenshot(`badge-hint-tab-button-icon-top`));
+    });
+
+    test('tab button icon bottom, should not have visual regressions', async ({ page }) => {
```

Comment:
```suggestion
    test('should not have visual regressions when icon is at the bottom', async ({ page }) => {
```
This follows the title standards for testing.

Reply:
Done.